# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/credentialmanagement/identity/WebIdentityChild.cpp
# Commit: 459739221a88
# Full Hash: 459739221a884b2b1c2adab42fd19fafe279bd20
# Author: Sandor Molnar <smolnar@mozilla.com>
# Date: 2025-06-05 04:07:37
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Revert "Bug 1969015, 1969017: apply code formatting via Lando" for causing build bustages
#   
#   This reverts commit ab681e36aeafc21ba62bb4680bdaa6206f02546d.
#   
#   Revert "Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz"
# ==============================================================================

diff -r fc9986230203 -r 459739221a88 dom/credentialmanagement/identity/WebIdentityChild.cpp
--- a/dom/credentialmanagement/identity/WebIdentityChild.cpp	Thu Jun 05 00:15:19 2025 +0300
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,23 +0,0 @@
-/* -*- Mode: C++; tab-width: 8; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
-/* vim: set ts=8 sts=2 et sw=2 tw=80: */
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-#include "mozilla/dom/WebIdentityChild.h"
-#include "mozilla/dom/WebIdentityHandler.h"
-
-namespace mozilla::dom {
-
-void WebIdentityChild::ActorDestroy(ActorDestroyReason why) {
-  if (mHandler) {
-    mHandler->ActorDestroyed();
-    mHandler = nullptr;
-  }
-}
-
-void WebIdentityChild::SetHandler(WebIdentityHandler* aHandler) {
-  mHandler = aHandler;
-}
-
-}  // namespace mozilla::dom